elastic/integrations
# [Logstash] Add guard against missing elements in `codec` payload

## Proposed commit message

Add guard against missing elements in `codec` payload
    
Due to a bug in Logstash (https://github.com/elastic/logstash/issues/16410), when configuring a codec with optional settings, the stats do not get generated, causing the data collection for codecs, and thus plugins to fail.

This commit adds a guard to the retrieval of `encode` and `decode` to ensure that the CEL input does not fail when encountering this issue.

## Checklist


## How to test this PR locally

Setup Logstash Integration to point to a working Logstash instance, the Logstash pipeline should include a codec that is configured with an optional parameter, eg

```
input { 
  stdin {
    codec => json { id => "json" }
  }
}
```

Navigate to the `Pipeline Details Overview` page, and verify that there is data for this pipeline. There should be entries in the `input` and `codec` sections of the page.

## Related issues

https://github.com/elastic/logstash/issues/16410
